neofetch

```diff
@@ -242,7 +242,13 @@ get_model() {
             fi
         ;;
 
-        "Mac OS X") model="$(sysctl -n hw.model)" ;;
+        "Mac OS X") 
+            if [[ $(kextstat | grep "FakeSMC") != "" ]]; then
```

Comment:
You mean `if [[ "$(kextstat | grep FakeSMC)" ]]; then`? Do I need to double quote "FakeSMC"?